core/domain/wipeout_service.py

```diff
@@ -197,10 +233,17 @@ def verify_user_deleted(pending_deletion_request):
         bool. True if all the models were correctly deleted, False otherwise.
     """
     user_id = pending_deletion_request.user_id
-    return all((
-        _verify_basic_models_deleted(user_id),
-        _verify_activity_models_deleted(user_id),
-    ))
+    role = pending_deletion_request.role
+    if role == feconf.ROLE_ID_LEARNER:
+        return all((
+            _verify_profile_related_models_are_deleted(user_id),
+            _verify_activity_models_deleted(user_id),
+        ))
+    else:
+        return all((
+            _verify_basic_models_deleted(user_id),
+            _verify_activity_models_deleted(user_id),
+        ))
```

Comment:
No, lets check everythin exactly the same as with regular user.
```suggestion
    return all((
        _verify_basic_models_deleted(user_id),
        _verify_activity_models_deleted(user_id),
    ))
```

Reply:
I forgot to fix that earlier as you had pointed out. Done now.